tools/online_editor/styles/content.css

```diff
@@ -26,12 +27,6 @@
   align-items: stretch;
 }
 
-.editor .container .tab-bar {
-  flex-shrink: 1;
-  flex-grow: 1;
-  background-color: pink;
-}
-
```

Comment:
This seems unrelated to the extraction of defaults into variables? Is this intentionally part of this commit?